R/compare-opts.R

```diff
@@ -2,10 +2,13 @@ compare_opts <- function(...,
                          tolerance = NULL,
                          max_diffs = if (in_ci()) Inf else 10,
                          ignore_srcref = TRUE,
-                         ignore_attr = FALSE,
+                         ignore_attr = "waldo_opts",
```

Comment:
Docs probably need to be updated to say that if you specify this yourself, you might need to include `waldo_opts`.

Reply:
Done